pkg/ocmcontainer/envs.go

```diff
@@ -0,0 +1 @@
+package ocmcontainer
```

Comment:
I might have missed it, but what is this file doing?

Reply:
Nothing when you commented.  It's handling environment variables now. :D